.eslintrc.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   root: true,
-  extends: "babel",
+  parser: "lightscript-eslint",
```

Comment:
Yes, if js it just calls eslint. But with the babel preset, it tries to import `babel-eslint` which doesn't exist when you do a clean install.